package trycatch;

public class ExTest {
	public static void main(String[] args) {

		int i = 50;
		int j = 0;
		int data;
		try {
			int x= i/j;
		} catch (Exception e1) {
			// TODO Auto-generated catch block
			e1.printStackTrace();
		}
		System.out.println("Welcome..!!");

	}
}
